tests/integration/test_storage_rabbitmq/test.py

```diff
@@ -3549,3 +3549,6 @@ def test_attach_broken_table(rabbitmq_cluster):
     assert "CANNOT_CONNECT_RABBITMQ" in error
     error = instance.query_and_get_error("INSERT INTO rabbit_queue VALUES ('test')")
     assert "CANNOT_CONNECT_RABBITMQ" in error
+
+
+# TODO: add a test
```

Comment:
Missing a clarification on which test should be added.

Reply:
This is just a reminder to me to add a test before merging this PR, was not planning to leave this comment here without adding a test.